### ISSUE TITLE
language selection improvements

### ISSUE THREAD

**Issue by severak** [elk-zone/elk#967]
Hi, as person who is switching between two languages (Czech and English) on my account I have two minor ideas how to improve post language selector:

- it should remember last selected language and for new posts use this
- it should be seen which language is currently chosen without going to dropdown (I would use language code for this purpose - e.g. show CZ or EN next to language chooser)

**Comment by alper** [comment]
Not to mention #1550

**Comment by kimusan** [comment]
I agree with this. Maybe even make it possible to limit the list of languages to just the one the use would use. I switch between 3–4 languages and really don't need African or Asian languages in my list.

**Comment by zordsdavini** [comment]
there should be possibility to add manual code as all languages never will be in the list. For example I write in Samogitian (sgs)
